static/js/reactions.js

```diff
@@ -78,16 +78,22 @@ function filter_emojis() {
     }
 }
 
-function maybe_select_emoji(e) {
+function get_emoji_from_index(index) {
     var reaction_list = $(".reaction-popover-reaction");
-    if (e.keyCode === 13) {
+    var reaction_show_list = reaction_list.filter(function () {
```

Comment:
This seems inefficient to do on every keystroke.  Can we refactor the code here so that we build up `reaction_show_list` only when we either render the widget whole or apply the filter?